components/list/list.js

```diff
@@ -64,6 +64,12 @@ class List extends LitElement {
 		`;
 	}
 
+	getListSize() {
```

Comment:
Add this method to the readme so people know about it 🍰 A test or two for it wouldn't be a terrible idea

Reply:
I also think this should be called `getListItemCount`.